logger.go

```diff
@@ -109,22 +103,61 @@ func createSentryHook(sentryDsn string) logrus.Hook {
 }
 
 /*
+getLevelFromEnv parses log level from environment variable LOGGER_LEVEL and apply sensible defaults when the value is
+absent or invalid.
+
+When LOGGER_LEVEL is not defined we use DebugLevel, which is convenient for development
+
+When LOGGER_LEVEL is invalid we use InfoLevel, which is convenient for production, and log a warning to help fix
+the situation
+
+Note: we do not use logrus.ParseLevel because we want to exclude warn, fatal and panic which are not a part of cp
+conventions, and we need to have error messages consistent with what's actually possible.
+ */
+func getLevelFromEnv() (logrus.Level, error) {
```

Comment:
I would give a function in parameter to get the env var: `func(string) string`.
Like that, we could pass `os.Getenv`, or something different like `viper.GetString`